scripts/gamecache/github_integration.py

```
@@ -67,11 +67,18 @@ def _make_http_delete(
     """Make HTTP DELETE request."""
     if headers is None:
         headers = {}
-
     try:
         _make_http_request(url, 'DELETE', None, headers, timeout)
         return True
-    except Exception:
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)